.azure/azure-nightly-template-windows.yml

```diff
@@ -43,7 +43,7 @@ jobs:
   - powershell: |
       $env:STACK_ROOT = "$(Build.SourcesDirectory)\.stack-root"
       $env:PATH += ";$env:HOME\.local\bin"
-      stack etc/scripts/release.hs --arch=i386 build
+      stack etc/scripts/release.hs --arch=x86_64 build
```

Comment:
It looks like this is now building 64-bit _twice_ (look two lines down), instead of building 32-bit and then 64-bit.

Reply:
Doh, good catch, pushing a fix.

Reply:
Does this mean we're going to stop building and distributing 32-bit `stack` bindists?

Reply:
Yes. Recent CI failures when upgrading to GHC 8.6.5 indicate that there are GHC bugs which will cause 32 bit Stack to crash